p2p/discovery/distance.go

```diff
@@ -24,8 +24,8 @@ func logdist(a, b common.Hash) int {
 	return len(a)*8 - lz
 }
 
-// distcmp compares the distances a->target and b->target.
-// Returns -1 if a is closer to target, 1 if b is closer to target
+// distcmp compares the distances a->to and b->to.
```

Comment:
why do you change target to to?  less meaningful and more confused for me

Reply:
target is confuse between network target and find node target. so change it. to means network target,